10_datastructures_lab/02.py

```diff
@@ -1,5 +1,28 @@
-"""
-Write a program that prints only unique values
-from sys.argv (without program name)
-"""
-
+"""
+The program receives input file containing a word in each line.
+Prints all anagrams in the same line
+"""
+
+import sys
+from collections import Counter
+
+if (len(sys.argv) != 2):
+    exit("Usage: 02.py <fileName>")
+
+fileName = sys.argv[1]
+
+repeats = {}

```

Comment:
A defaultdict can work better here. If `repeats = defaultdict(list)`. With that in place you won't need the if/else because values not yet created will automatically return a new empty array